### PR TITLE
Generated 0.2.1 install

### PR DESCRIPTION
**Pull request by ellis-bigelow**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/529)

### CODE REVIEW

File: config/default/configmap/inferenceservice.yaml [hunk 1]
```diff
@@ -30,23 +30,23 @@ data:
...
+               "0.2.1"
             ]
         },
         "xgboost": {
             "image": "gcr.io/kfserving/xgbserver",
-            "defaultImageVersion": "0.2.0",
+            "defaultImageVersion": "0.2.1",
             "allowedImageVersions": [
-               "0.2.0"
+               "0.2.1"
```
Comment (yuzisun) [inline config/default/configmap/inferenceservice.yaml]: some spaces here?

File: pkg/apis/serving/v1alpha2/inferenceservice_defaults_test.go [hunk 2]
```diff
@@ -168,7 +168,7 @@ func TestXGBoostDefaults(t *testing.T) {
 	g.Expect(isvc.Spec.Default.Predictor.XGBoost.Resources.Requests[v1.ResourceMemory]).To(gomega.Equal(defaultResource[v1.ResourceMemory]))
 	g.Expect(isvc.Spec.Default.Predictor.XGBoost.Resources.Limits[v1.ResourceCPU]).To(gomega.Equal(defaultResource[v1.ResourceCPU]))
 	g.Expect(isvc.Spec.Default.Predictor.XGBoost.Resources.Limits[v1.ResourceMemory]).To(gomega.Equal(defaultResource[v1.ResourceMemory]))
-	g.Expect(isvc.Spec.Canary.Predictor.XGBoost.RuntimeVersion).To(gomega.Equal("0.2.0"))
+	g.Expect(isvc.Spec.Canary.Predictor.XGBoost.RuntimeVersion).To(gomega.Equal("0.2.1"))
```
Comment (pugangxa) [inline pkg/apis/serving/v1alpha2/inferenceservice_defaults_test.go]: Can we define a constant for the version number to avoid update so much places?

Comment (animeshsingh) [reply pkg/apis/serving/v1alpha2/inferenceservice_defaults_test.go]: Agreed

File: pkg/apis/serving/v1alpha2/inferenceservice_types.go [hunk 3]
```diff
@@ -193,7 +192,7 @@ type SKLearnSpec struct {
 type ONNXSpec struct {
 	// The location of the trained model
 	StorageURI string `json:"storageUri"`
-	// Allowed runtime versions are [v0.5.0, latest] and defaults to the version specified in inferenceservice config map
+	// Allowed runtime versions are [v0.5.0, latest] and defaults to the version specified in the inferenceservice config map
```
Comment (yuzisun) [inline pkg/apis/serving/v1alpha2/inferenceservice_types.go]: remove versions here as well

File: pkg/apis/serving/v1alpha2/inferenceservice_types.go [hunk 4]
```diff
@@ -163,7 +162,7 @@ type TensorflowSpec struct {
 type TensorRTSpec struct {
 	// The location of the trained model
 	StorageURI string `json:"storageUri"`
-	// Allowed runtime versions are [19.05-py3] and defaults to the version specified in inferenceservice config map
+	// Allowed runtime versions are [19.05-py3] and defaults to the version specified in the inferenceservice config map
```
Comment (yuzisun) [inline pkg/apis/serving/v1alpha2/inferenceservice_types.go]: remove versions

**Pull request merged**